_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: seo drupal drupal8
+summary: SEO trends 2017 and Drupal 8 as the best option for your long-term marketing campaign.
+featured_image: "/assets/img/blog/supercharged-seo-with-drupal-8.jpg"
+featured_image_alt: "Supercharged SEO with Drupal 8"
+---
+
+Here at Savas Labs, when it comes to developing systems, our focus is not only to code according to best practices, but also deliver a solid marketing vehicle to our clients and partners. Let’s be honest here - perfectly coded, a pretty looking website will be of little use if it doesn’t produce leads / facilitate conversions or generate revenue! So how do we achieve those goals? We get there by balancing website objectives while giving priority to lead generation via SEO. The more quality traffic that comes to the website - the more conversions we can achieve. It’s that simple!
+
+There are multiple ways of generating traffic to a website. The most popular are SEO (search engine optimization) and PPC (Pay Per Click). Both bring traffic through search engines. The difference between the two is that SEO brings long-term results boosting organic traffic while PPC helps marketers achieve short-term goals by gaining instant exposure throughout the duration of the ad campaign.
+
+In this post I’ll share some insight about current trends of SEO. I’ll also describe new features of  Drupal 8 that make it the most SEO-friendly content management framework available today.
+
+Let’s take a look at what exactly Search Engine Optimization is.
+
+## What is SEO?
+
+Search Engine Optimization (SEO) is a marketing discipline focused on optimizing a website’s architecture and content so that it performs well (read “ranks high”) in organic search engine results.
+
+Search engines (Google, Bing, etc.) are changing their search algorithms many times throughout the year. There are over 200 ranking factors that become updated with every algorithmic change. Having said that, we begin to realize that there are many techniques that website owners can use to optimize their digital property for search engine consumption. So where do we start? How do we know what efforts will bring us the best ROI and let our marketers do their job effectively in the long run?
+
```

Comment:
Instead of "Having said that" Perhaps "Given this reality, there are many techniques..."